src/controller/python/chip/clusters/attribute.cpp

```diff
@@ -308,10 +316,11 @@ void pychip_ReadClient_Abort(ReadClient * apReadClient, ReadClientCallback * apC
 
 chip::ChipError::StorageType pychip_ReadClient_ReadAttributes(void * appContext, ReadClient ** pReadClient,
                                                               ReadClientCallback ** pCallback, DeviceProxy * device,
-                                                              bool isSubscription, uint32_t minInterval, uint32_t maxInterval,
-                                                              size_t n, ...)
+                                                              uint8_t * readParamsBuf, size_t n, ...)
 {
-    CHIP_ERROR err = CHIP_NO_ERROR;
+    CHIP_ERROR err                 = CHIP_NO_ERROR;
+    PyReadAttributeParams pyParams = {};
+    memcpy(&pyParams, readParamsBuf, sizeof(pyParams));
```

Comment:
Why do we need this memcpy?  Is it because there is a chance readParamsBuf may not be aligned?

Reply:
Yes, it should not crash but I would avoid that corner case.